build.sbt

```diff
@@ -36,7 +36,10 @@ val extraDependencies = Seq(
   // Although breeze 1.2 is already provided by Spark, this is needed for Azure Synapse Spark 3.2 pools.
   // Otherwise a NoSuchMethodError will be thrown by interpretability code. This problem only happens
   // to Azure Synapse Spark 3.2 pools.
-  "org.scalanlp" %% "breeze" % "1.2"
+  "org.scalanlp" %% "breeze" % "1.2",
+  "com.typesafe.play" %% "play" % "2.8.8",
```

Comment:
please remove unused deps here, (ideally try to remove all if possible as each dep can make it harder to install on synapse)

Reply:
Done